### DISCUSSION TITLE
Jason Liu

### DISCUSSION THREAD

**Discussion by hipstersmoothie** [devtools-fm/devtools.fm#304] [Announcements]
- https://x.com/jxnlco
- https://jxnl.co/
- https://github.com/jxnl/instructor

## Questions

- [x] What exactly is an AI engineer?
- [x] Do you think these people should just be using python?
- [x] What things do you think a dev needs to learn about to be a good AI engineer? Theres lots of new acronyms popping up and I don't what to follow!
- [x] What do you think most people get wrong when trying to create things with LLMs?
- [x] What are some of the hard things about deploying LLMs to production?
- [x] What do you do right now? How exactly do you help companies be better at LLMs?
- [x] You've said that you don't really need devtools for LLMs and you just need pydantic. Do you really thinks that's true? Or is there room?
- [x] What are the coolest non-chatbot products/projects you've seen/can imagine/are excited by?

### Instructor

- [x] What is instructor? How does it help me with LLMs?
- [x] What is it actually doing to turn the LLMs response into the desired output?
- [ ] ~How does pydantic play into this?~

### Future

- [x] As models get better do you think prompt engineering will less important?
- [x] If you could have any one feature request prioritized by all the LLM providers, what would it be?
- [ ] ~What do you think the future of software interfaces looks like? Are we all just chatting with chat bots?~

**Comment by jxnl** [comment]
[betterdictation.com](https://betterdictation.com/) code JASON20
[cursor.ai](https://cursor.sh/)
https://one-sec.app/

**Comment by hipstersmoothie** [comment]
http://v0.dev
https://github.com/ChrisBuilds/terminaltexteffects

**Comment by just-be-dev** [comment]
- https://github.com/dosco/llm-client
- https://github.com/ThousandBirdsInc/chidori
